src/main/resources/org/sonatype/gradle/plugins/scan/ossindex/header.txt

```diff
@@ -0,0 +1,15 @@
+ ________  ________  ________  ________  ___       _______           ________  ________  ________  ________
```

Comment:
I'd prefer this file to be called "banner.txt" instead for two reasons:
1. The class printing this is `BannerUtils`
2. The file we use internally at Sonatype for the license header is `header.txt` (to use a Maven plugin I believe), so this file might create confusions.

Reply:
> I'd prefer this file to be called "banner.txt" instead for two reasons:
> 
> 1. The class printing this is `BannerUtils`
> 2. The file we use internally at Sonatype for the license header is `header.txt` (to use a Maven plugin I believe), so this file might create confusions.

Done. 36701e65